CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ description: |-
 ---
 
 # VKCS Provider's changelog
+#### v0.7.1 (unreleased)
+- Loadbalancer no longer returns SG of vip port
```

Comment:
It's better to follow common pattern and start changelog messages with a verb, also it seems to be kind of unclear, may be, it's better to describe the reason for change, not the change itself, as it's hard to understand why it's needed.
For example, "Fix reading load balancer resource in error state with deleted VIP port".

Reply:
By the way, commit message isn't clear as well, "fix linter and changelog" should be removed, it seems like they were added during the squash, commit title may be the same as the changelog message, and the commit description may consist of info about the change itself and removed reading of security groups on VIP port.

Reply:
i have plagiarized second variant, thanks)